packages/fx-core/resource/package.nls.json

```diff
@@ -673,7 +673,7 @@
   "error.aad.manifest.AADManifestIssues": "AAD manifest has following issues that may potentially break the Teams App:\n",
   "error.aad.manifest.UnknownResourceAppId": "Unknown resourceAppId %s",
   "error.aad.manifest.UnknownResourceAccessType": "Unknown resourceAccess: %s",
-  "error.aad.manifest.UnknownResourceAccessId": "Unknown resourceAccess id: %s",
+  "error.aad.manifest.UnknownResourceAccessId": "Unknown resourceAccess id: %s, if you're using human-readable string as resourceAccess id, please try to use uuid instead.",
```

Comment:
do we have more suitable wording for "human readable string"?

Reply:
how about scope string?